vllm/entrypoints/openai/serving_transcription.py

```diff
@@ -292,7 +298,12 @@ async def create_transcription(
             # TODO: Use a vllm-specific Validation Error
             return self.create_error_response(str(e))
 
-        # TODO(rob): figure out a way to pipe streaming in.
+        if request.stream:
+            return self.transcription_stream_generator(request,
```

Comment:
I think we should use `StreamingResponse` and pass the `generator` to `StreamingResponse`. Check out the `/v1/competions` code for an example of this.

Reply:
already done in `api_server`